### PR TITLE
Fairground organ style + 2

### PR DESCRIPTION
**Pull request by karst**
This PR will add the first songs for the replacement of the original fairground organ style.

It will also create an entirely new fairground organ style, new music for our ears! I imagine a number 3 will come and maybe more in the future :)

Fairground organ style 2 consists of 60 minutes and 30 seconds of new music from the original RCT organ.

photo_2023-04-01_00-14-13

### CODE REVIEW

**Comment by karst** [comment]
What's left to do is figuring out all the names for the songs. Hopefully a friend of Herman can help me out. And otherwise the community.

**Comment by karst** [comment]
Audio is quieter than the original master, do I need to turn it up a bit? Normalize the audio at least or should i push it more against the ceiling similar to the original master?

**Comment by Gymnasiast** [comment]
Ok, if I understand correctly, this is just a selection of the songs that the organ has, right? I’m missing a few that I saw on the Tom Gavioli channel, but if I read the description correctly, those could be in part 3?

1.flac seems to have some talking in the background (around 0:10). 3.flac and 11.flac have this too.

10.flag is “Unter dem Doppeladler” from Josef Franz Wagner.

17.flac is the Colonel Bogey March, but it looks like that one was already identified. 

Regrettably, I have not been able to identify any more.

**Comment by karst** [comment]
What should I do with the recordings that have a light background talk? I thought i'd leave it in since it'd mix with the other noises of the park anyhow and it wouldn't be noticable. But if it's too much of an annoyance i'll leave the songs out. I'll make sure to edit 10 to have the right name.

And you are indeed correct, these are from three of the several books the organ has.

**Review by tupaschoal — COMMENTED** [review]
> What should I do with the recordings that have a light background talk?

On my lousy speakers I could barely distinguish it.

I don't know how to help identifying, so I used google voice assistant and had no luck other than identifying 10 and 17 (which were already identified)

**Comment by karst** [comment]
Thanks for checking! I hope once we've got this merged the community might let us know.

**Review by tupaschoal — APPROVED** [review]
Not sure if you intend to add more stuff here, but it looks good as is for me.

**Comment by karst** [comment]
I don't have any more recordings. I only have the two from the original + this playlist. In next recording session I might make a third playlist. Should help a lot of with making for different tunes in the soundscape of the park.

File: openrct2.music.cover.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            "id": "rct2.music.fairground",
+            "samples": [
+                "replacements/openrct2.music.fairground/das_alpenhorn.flac",
+                "replacements/openrct2.music.fairground/slavonic_dance.flac"
+            ]
+        }
+    ],
+    "strings": {
+        "name": {
+            "en-GB": "OpenRCT2 Music (Cover)"
```
Comment (tupaschoal) [inline openrct2.music.cover.json:21]:
```suggestion
            "en-GB": "OpenRCT2 Music (Cover)",
            "pt-BR": "Músicas do OpenRCT2 (Cover)"
```

File: openrct2.music.cover.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        },
+        "description": {
+            "en-GB": "Covers of public domain ride music for OpenRCT2."
```
Comment (tupaschoal) [inline openrct2.music.cover.json]:
```suggestion
            "en-GB": "Covers of public domain ride music for OpenRCT2.",
            "pt-BR": "Covers de músicas de atração de domínio público para o OpenRCT2"
```

File: openrct2.music.cover.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "id": "openrct2.music.cover",
+    "version": "1.0",
+    "objects": [
+        {
+            "id": "rct2.music.fairground",
+            "samples": [
+                "replacements/openrct2.music.fairground/das_alpenhorn.flac",
+                "replacements/openrct2.music.fairground/slavonic_dance.flac"
```
Comment (karst) [inline openrct2.music.cover.json]: Move to cover folder

Comment (karst) [inline openrct2.music.cover.json]: idem

File: openrct2.music.cover.json [hunk 4]
```diff
@@ -12,7 +12,8 @@
     ],
     "strings": {
         "name": {
-            "en-GB": "OpenRCT2 Music (Cover)"
+            "en-GB": "OpenRCT2 Music (Cover)",
```
Comment (karst) [inline openrct2.music.cover.json]: `"nl-NL": "OpenRCT2 Muziek (Cover)",`

**Review by IntelOrca — APPROVED** [review]

**Pull request merged**